packages/docs/celo-codebase/protocol/release-gold/README.md

```diff
@@ -0,0 +1,100 @@
+
+## Use Cases for ReleaseGold
+
+Two anticipated use cases for ReleaseGold contracts are for “holders” and “earners”. Note that these are not specified in ReleaseGold explicitly, rather they represent sample personas of beneficiaries for which ReleaseGold was designed.
+
+In the “holder” case, a recipient has purchased or been awarded an amount of cGLD, but is subject to a distribution schedule limiting the amount of cGLD that can be liquidated at any given time. These recipients will likely be able to validate and vote with the full ReleaseGold balance, and also are not subject to the contract’s revocation by another party (eg. an employer).
+
+In the “earner” case, a grant recipient has entered a legal contract wherein an exchange of services earns them an amount of cGLD over a releasing, or vesting, schedule. These grants are characterized by extra restrictions because the total grant amount is still being *earned*. The ReleaseGold balance cannot be used for running a validator, but it can be used to vote for validators and governance proposals on the Celo network. Additionally, these contracts are revocable and subject to the `liquidityProvision` flag, which prevents cGLD distribution when markets are incapable of absorbing additional cGLD without significant slippage.
+
+## Release Schedule
+
+In ReleaseGold smart contracts, a fixed amount of cGLD becomes accessible to the `beneficiary` over time.
+
+The following arguments specify a ReleaseGold smart contract schedule:
+
+- `releasePeriod` - the frequency, in seconds, at which cGLD is released
+    - Some common values: monthly (2628000), every 3 months (7884000)
+- `amountReleasedPerPeriod` - the amount of cGLD to be released each `releasePeriod`
+- `numReleasePeriods` - the number of `releasePeriods` in which cGLD will be released
+- `releaseCliff` - the time at which the release cliff expires.
+
+The total balance for the ReleaseGold account can be determined by multiplying the `numReleasePeriods` by `amountReleasedPerPeriod`.
+
+Similar to vesting-type schedules with cliffs used for other assets, ReleaseGold allows for a `releaseCliff` (expressed in seconds) before which the released cGLD cannot be withdrawn by its beneficiary. A common value for this is `31536000`, which is 1 year.
+
+## Released and Unreleased cGLD
+
+In deployed ReleaseGold accounts, you can conceptually think of cGLD in two states -- released, and unreleased. There are other states including locked, but for the purposes of the contract, these are the two primary states to consider.
+
+Released cGLD can be withdrawn to the `beneficiary` where they can be used freely. Unreleased cGLD comes with some restrictions. Foremost, it cannot be withdrawn by the beneficiary. If `canVote` and `canValidate` are set to false, the beneficiary cannot vote or validate, respectively.
```

Comment:
```suggestion
Released cGLD can be withdrawn to the `beneficiary` where it can be used freely. Unreleased cGLD comes with some restrictions. Foremost, it cannot be withdrawn by the beneficiary. If `canVote` and `canValidate` are set to false, the beneficiary cannot vote or validate, respectively.
```